src/js/views/patients/worklist/worklist_views.js

```diff
@@ -213,6 +213,8 @@ const ListView = CollectionView.extend({
         return matcher.test(searchString);
       });
     });
+
+    this.state.set('lastSelectedId', null);
```

Comment:
Clears the `state.lastSelectedId` value when the `Find in list...` filtering is applied to the list.